lib/glific/partners.ex

```diff
@@ -1308,4 +1309,41 @@ defmodule Glific.Partners do
         |> Repo.update()
     end
   end
+
+  @doc """
+  Update report_frequency field in settings column
+  """
+  @spec update_report_frequency(non_neg_integer(), String.t()) ::
+          tuple()
+  def update_report_frequency(org_id, frequency \\ "MONTHLY") do
```

Comment:
why is this function needed? can we use update_organization?

Reply:
We can also use update_organization to update the setting column but then first we'll have to fetch the existing setting and then update the map and pass it to the changeset.

In this function we are directly updating the setting column without need to fetch it in separate query

Reply:
will it append the existing setting or replace it entirely. Also we need to make it a generic function something like update setting so same code can be reused to update differnt key value pair in settings

Reply:
It will update the existing setting. If the key is not present in the setting then it will create one and if the key is already there then it will just update the value of the key